Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM usdotfhwastoldev/carma-base:develop AS base_image
+ARG DOCKER_ORG=usdotfhwastoldev
+ARG DOCKER_TAG=develop
+FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
 
 FROM base_image AS build
-
+ARG GIT_BRANCH=develop
```

Comment:
Or if this PR is only addressing the fix from an automated CI standpoint, we can create a separate story (or multiple) to fix it from a user standpoint.

Reply:
I think they new arguments in the docker file will largely replace build-image.sh functionality that is responsible for setting org/tag for images.

Reply:
I am okay with merging this since that build-images.sh script needed manual edits to work correctly anyways (such as checkout scripts or Dockerfile edits). 
I do agree we can create new story to modify pass in these arguments based on what `build-script.sh` sets the metada.